activerecord/lib/active_record/enum.rb

```diff
@@ -218,6 +218,13 @@ def assert_valid_enum_definition_values(values)
           MSG
           raise ArgumentError, error_message
         end
+
+        if values.is_a?(Hash) && values.keys.any?(&:empty?)
```

Comment:
This doesn't catch the case where `values` is an array (`enum col: ['']`). Maybe we should use `if values.to_a.flatten.any?(&:blank?)` to cover both cases? This would subsequently check values not be blank but I'm not sure if that's a bad thing.

Reply:
Good idea, but instead of `if values.to_a.flatten.any?(&:blank?)` I replaced the condition with `values.is_a?(Hash) && values.keys.any?(&:blank?) || values.is_a?(Array) && values.any?(&:blank?)`,
because we don't want to generate this error in cases someone has a perfectly valid `enum :status, { active: 1, inactive: nil }`